app/controllers/users/backup_code_setup_controller.rb

```diff
@@ -97,7 +97,10 @@ def track_backup_codes_created
         enabled_mfa_methods_count: mfa_user.enabled_mfa_methods_count,
         in_account_creation_flow: in_account_creation_flow?,
       )
-      Funnel::Registration::AddMfa.call(current_user.id, 'backup_codes', analytics)
+      Funnel::Registration::AddMfa.call(
+        current_user.id, 'backup_codes', analytics,
+        threatmetrix_attrs
```

Comment:
Nit: Would suggest putting all arguments on the same line, or a single argument per line. I find mixed number of arguments per line to be difficult to scan.

```suggestion
        current_user.id,
        'backup_codes',
        analytics,
        threatmetrix_attrs,
```

```suggestion
        current_user.id, 'backup_codes', analytics, threatmetrix_attrs,
```